index.bs

```diff
@@ -1285,6 +1333,10 @@ this is 0.
 <dfn>Default aggregation coordinator</dfn> is the [=aggregation coordinator=] that controls how to
 obtain the public key for encrypting an [=aggregatable report=] by default.
 
+<dfn>Verification tokens per trigger</dfn> is a positive integer that controls the number of
```

Comment:
Should this be a vendor-specified value, or do we want it to be a constant mandated by the spec?

Reply:
There is a utility/performance trade off associated with this value which I think could justify making it a vendor-specified value.

More details in this issue: https://github.com/WICG/attribution-reporting-api/issues/875